bluetooth/src/commonMain/kotlin/Characteristic.kt

```diff
@@ -109,10 +109,31 @@ open class Characteristic(val wrapper: CharacteristicWrapper, initialValue: Byte
     override fun getUpdatedValue(): ByteArray? {
         return wrapper.value?.asBytes
     }
+
+    fun hasProperty(property: CharacteristicProperties) = hasProperties(listOf(property))
+
+    private fun hasProperties(properties: List<CharacteristicProperties>) = wrapper
```

Comment:
Did you consider using `Set` or is there a good reason to use `List`?

Reply:
It's private and just wrapper over `vararg` below.

Reply:
Oke 🙂